Apps/Contoso.Forms.Demo/Contoso.Forms.Demo.UWP/Contoso.Forms.Demo.UWP.csproj

```diff
@@ -135,10 +135,10 @@
     </ProjectReference>
   </ItemGroup>
   <ItemGroup>
-    <PackageReference Include="Microsoft.AppCenter.Analytics" Version="3.4.3" />
-    <PackageReference Include="Microsoft.AppCenter.Crashes" Version="3.4.3" />
-    <PackageReference Include="Microsoft.AppCenter.Distribute" Version="3.4.3" />
-    <PackageReference Include="Xamarin.Forms" Version="4.8.0.1451" />
+    <PackageReference Include="Microsoft.AppCenter.Analytics" Version="4.0.0" />
+    <PackageReference Include="Microsoft.AppCenter.Crashes" Version="4.0.0" />
+    <PackageReference Include="Microsoft.AppCenter.Distribute" Version="4.0.0" />
+    <PackageReference Include="Xamarin.Forms" Version="4.8.0.1687" />
```

Comment:
There has been an issue with AC Build with versions of Xamarin.Forms > `4.8.0.1451`, I suggest keeping it at `4.8.0.1451` up until next release (at least), in order to avoid those problem on AC Build.

Reply:
Do we need to keep version at 4.8.0.1451 about  these four files? 
Apps\Contoso.Forms.Demo\Contoso.Forms.Demo\Contoso.Forms.Demo.csproj
Apps\Contoso.Forms.Demo\Contoso.Forms.Demo.Droid\Contoso.Forms.Demo.Droid.csproj
Apps\Contoso.Forms.Demo\Contoso.Forms.Demo.iOS\Contoso.Forms.Demo.iOS.csproj
Apps\Contoso.Forms.Demo\Contoso.Forms.Demo.UWP\Contoso.Forms.Demo.UWP.csproj

Reply:
Yup. That's my suggestion

Reply:
Thank you for your help. I updated it.